dev_suites/dev_requirements/requirements_suite.rb

```diff
@@ -0,0 +1,40 @@
+module RequirementsSuite
+  class Suite < Inferno::TestSuite
+    title 'Requirements Suite'
+    id :ig_requirements
+    description 'Suite Description'
+
+    verifies_requirements 'sample-criteria-proposal@1', 'sample-criteria-proposal@2',
```

Comment:
Shouldn't higher-level groups automatically inherit all of the requirements their children verify?

Reply:
yeah, makes sense.

Reply:
I don't think they should automatically inherit. Example illustrating why:


- Group 1
  - Test A -> requirement 1
  - Test B (optional) -> requirement 2

An execution of Group 1 where Test A succeeds and Test B fails would result in Group 1 succeeding but not in requirement 2 being met. I think we want to keep the property that if a runnable succeeds then that implies that all its associated requirements are met. Inheritance would break this property because Group 1 succeeded but requirement 2 is not met, so Group 1 cannot inherit the requirements from its children.

Its possible that I'm wrong about wanting a success to indicate requirements are met, but my gut says that's important, so if you have different thoughts, I'd want to talk through it further.